sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/AppConfigurationTest.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.AppConfiguration.Models;
+using NUnit.Framework;
+using Sku = Azure.ResourceManager.AppConfiguration.Models.Sku;
+
+namespace Azure.ResourceManager.AppConfiguration.Tests
+{
+    public class AppConfigurationTest : AppConfigurationClientBase
+    {
+        public AppConfigurationTest(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        [SetUp]
+        public void ClearChallengeCacheforRecord()
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                Initialize();
+            }
+        }
+
+        [TearDown]
+        public async Task CleanupResourceGroup()
+        {
+            await CleanupResourceGroupsAsync();
+        }
+
+        [Test]
+        public async Task Test_AppConfiguration_list_Key_Values()
+        {
+            var resourceGroup = Recording.GenerateAssetName(ResourceGroupPrefix);
+            await Helper.TryRegisterResourceGroupAsync(ResourceGroupsOperations, AZURE_LOCATION, resourceGroup);
+            //create configuration
+            var configuration_Store_Name = Recording.GenerateAssetName("configuration");
+            var configurationCreateResult = await ConfigurationStoresOperations.StartCreateAsync(resourceGroup, configuration_Store_Name, new ConfigurationStore("westus", new Sku("Standard")));
+            var configCreate = (await WaitForCompletionAsync(configurationCreateResult)).Value;
+
+            //list configuration
+            var configListResult = ConfigurationStoresOperations.ListKeysAsync(resourceGroup, configuration_Store_Name);
+            var conList = await configListResult.ToEnumerableAsync();
+            //# ConfigurationStores_ListKeys[post]
+            //    keys = list(self.mgmt_client.configuration_stores.list_keys(resource_group.name, CONFIGURATION_STORE_NAME))
+            var configRegenerateResult = ConfigurationStoresOperations.RegenerateKeyAsync(resourceGroup, configuration_Store_Name,new RegenerateKeyParameters(conList.First().Id));
+
+            //if self.is_live:
+            //    # create key-value
+            //    self.create_kv(key.connection_string)
+
+            //create Key
+            //var addConfigurationsetting = await PrivateEndpointConnectionsClient
+            //[skip]
+            //var listkeyvalueResult = await ConfigurationStoresClient.ListKeyValueAsync(resourceGroup, CONFIGURATION_STORE_NAME,KEY,LABEL);
+            //# ConfigurationStores_ListKeyValue[post]
+            //    BODY = {
+            //        "key": KEY,
+            //  "label": LABEL
+            //}
+            //    result = self.mgmt_client.configuration_stores.list_key_value(resource_group.name, CONFIGURATION_STORE_NAME, BODY)
```

Comment:
Should we keep these commented lines? Let's talk in the meeting

Reply:
ok,thanks